src/main/java/org/reaktivity/nukleus/tcp/internal/InternalSystemProperty.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2007-2016, Kaazing Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.reaktivity.nukleus.tcp.internal;
+
+import java.util.Properties;
+import java.util.function.IntSupplier;
+
+public enum InternalSystemProperty
+{
+
+    // Maximum window size for nuklei writing data to the tcp nukleus
+    WINDOW_SIZE("nukleus.tcp.window.size", "8192"),
+
+    MAXIMUM_MESSAGE_SIZE("nukleus.tcp.maximum.message.size", "8192"),
```

Comment:
Why is `MAXIMUM_MESSAGE_SIZE` needed, independent from `WINDOW_SIZE`?
There is no concept of message in TCP since it is just a stream, there are just outstanding bytes still to arrive, indicated by window size, right?